<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrations</class>
 <widget class="QDialog" name="registrations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Регистрация</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="emailEdit">
          <property name="placeholderText">
           <string>Электронная почта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="telephEdit">
          <property name="placeholderText">
           <string>Номер телефона</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="placeholderText">
           <string>Имя пользователя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="licenseEdit">
          <property name="placeholderText">
           <string>Серия и номер В/У</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passEdit">
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Продолжить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
